python/ray/_private/workers/default_worker.py

```diff
@@ -204,6 +204,17 @@
     # for asyncio
     try_install_uvloop()
 
+    # If the worker setup function is configured, run it.
```

Comment:
why moving this

Reply:
[demo purpose] this is to run the setup hook before initing Ray (that is, creating a Node and talking to raylet). Used in `test_can_reuse_released_workers` to test the decoupling of worker creation vs the task request. I can remove it once we get a better way of testing this.

Reply:
reverted